library/std/src/sys/hermit/thread.rs

```diff
@@ -97,6 +102,9 @@ impl Thread {
     }
 }
 
+pub type Priority = ();
```

Comment:
These should be structs instead of type definitions in case this is implemented for the target in the future.

Reply:
Since this is a private type, I think it should be possible to change its type freely without any consequences to user code (alias or otherwise). However if it was re-exported as `pub` (per your other comment) then I agree it would be better as `struct Priority(())`

Reply:
Ah yeah, this is sys and not _actually_ public